user/articles/06-app-management/07-disaster-recovery.markdown

```diff
@@ -0,0 +1,154 @@
+| **3:00 pm**  | Everything is back to normal and now we're serving traffic through the DR environment. |
+
+
+
+###  **2. Once the disaster is no longer present**
+
+
+![Picture 2: Restoring from Disaster Recovery Environment to Production](../../images/dr-02.png)
+
+|              |                                                              |
+| ------------ | ------------------------------------------------------------ |
+| **4:00 pm**  | we are notified that the *eu-west2* region has been recovered and the incident is over, which means that we are ready to go back to our original Production environment |
+| **4:16 pm** | We'll perform a manual backup since there has been new user data generated during the period we were using the DR environment. |
+| **4:21 pm**  | We can use the manual backup we just generated to restore the Production environment and then change the Custom Domain. |
+
+Now we're back to serving traffic through the Production environment and our DR environment will stay running for when we need it again.
+
+
+
+## Initial Setup
+
+In order to start using the Disaster Recovery environment, it's important to review a few items that are needed to ensure optimum experience and achieve the least downtime possible. These steps can and should be taken before disaster even happens.
+
+### **1. Make sure the Disaster Recovery Environment has been created and that you have Admin access to both environments**
+
+There are certain actions that the Liferay Cloud team needs to perform before you can use this feature such as marking the **dr** environment as a production environment. Make sure to contact us to confirm that these configurations have been made.
+
+You must also be an Admin member on both environments.  
+
+![Picture 3](../../images/dr-03.png)
+
+### 2. Make sure the VPN has been configured in the DR environment**
+
+If you have VPN enabled on the Production environment, you need to make sure that has also been configured in the DR environment.
+
+![Picture 4](../../images/dr-04.png)
+
+
+### **3. Deploy the latest stable Build from Production to DR environment**
+
+Go to the Builds page and find the number of the latest build that was deployed to the Production environment. Select that build and deploy it to the DR environment.
+
+
+![Picture 5](../../images/dr-05.png)
+![Picture 6](../../images/dr-06.png)
+
+## Automatic Data Recovery Workflow
+
+The ability to recover the database and documents from a backup in an automatic way is not available yet.
+
+However, we will introduce a feature called *Auto Restore* which will allow users to set up a scheduler to automate the restore processes from one environment to the other.
+
+This will improve the overall time to recover DR environments and decrease the chances of human error.
+
+<u>The goal is to deliver this feature to customers in mid-Q4.</u>
```

Comment:
I have the impression this feature was already added. Please check with Igor Nazar and Zeno.

Reply:
Hi Eduardo and Bernard,
1. We should not put anything in the official documentation describing missing features or promising new features in the future. Docs should always describe the product as-is, not what it could be. 
2. We should minimize using any value judgments: "the best ever, easiest, and decreasing human error." That's for marketing and sales.
3. We're also moving towards a simpler writing style and format - no need to tell stories about cars and fixing tires for hot, cold and warm strategies. Same with the long timeline. 

Thanks.